src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -108,6 +111,15 @@ public ObservableList<Person> getPersonList() {
         return persons.asUnmodifiableObservableList();
     }
 
+    public ObservableList<Policy> getPolicyList() {
+        List<Policy> allPolicies = new ArrayList<>();
+        for (Person person : persons) {
+            PolicySet pset = person.getPolicySet();
+            allPolicies.addAll(pset.getAllPolicies());
```

Comment:
i think can just call allPolicies.addAll(pset) so you dont need to create a new function for PolicySet#getAllPolicies.

Reply:
this was implemented to get the entire set of policies from all persons. if we remove this method, i will need to loop through all persons object elsewhere to retrieve all pset before i can use addall(). I think this way it will be more readable